sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/AsymmetricEncryptionAlgorithm.java

```diff
@@ -29,10 +29,10 @@ protected AsymmetricEncryptionAlgorithm(String name) {
      * uses the specified {@link KeyPair} and the default {@link Provider} provider.
      *
      * @param keyPair The key pair to use.
-     * @return
-     * @throws InvalidKeyException
-     * @throws NoSuchAlgorithmException
-     * @throws NoSuchPaddingException
+     * @return abstract {@link ICryptoTransform}
+     * @throws InvalidKeyException when key is not valid
+     * @throws NoSuchAlgorithmException if algorithm is not found
+     * @throws NoSuchPaddingException if padding is set wrong
```

Comment:
This exception text should be improved after this is merged (by @g2vinay)

Reply:
@g2vinay if you share whatever text you want in there, I can make it part of this PR

Reply:
> This exception text should be improved after this is merged (by @g2vinay)
This is not public API, 
I am just planning to keep the desrcription part of javadoc as a comment, will do it in a follow up clean up PR. @vhvb1989